tests/integration/test_exodus.py

```diff
@@ -0,0 +1,98 @@
+import requests
```

Comment:
Please add requests into test-requirements.txt as well, as nothing guarantees that it's installed.